optimization/matRad_IpoptIterFunc.m

```diff
@@ -65,7 +65,8 @@
     hold on, grid on, grid minor,
     AxesInfigOpt = findall(figOpt,'type','axes');
 end
-
+% ensure to bring optimization window to front also for a re-optimization 
+figure(figOpt);
```

Comment:
Yes it does - the iterations plot will be shown on top. If the figure is closed then it will be re opened in the next iteration. Consequently during optimisation the plot is always visible.
Otherwise I had the problem when working solely with the GUI that the optimization window didn't show up in the front for a re-optimization. This I found very inconvenient as the user  has no clue what is going after hitting the optimization button. We could add a if statement to solve this. if deployed then force the window on the front else stick to the old behaviour.